Backpack/NavigationBar/Classes/BPKNavigationBar.m

```diff
@@ -346,10 +354,38 @@ - (UIViewController *_Nullable)findNearestViewController {
     return (UIViewController *)responder;
 }
 
+- (void)updateAppearance {
+    self.backgroundView.backgroundColor = self.backgroundViewColor;
+    
+    self.leftButton.linkContentColor = self.contentColor;
+    self.rightButton.linkContentColor = self.contentColor;
+    self.titleView.titleLabel.textColor = self.contentColor;
```

Comment:
This will change the behaviour for the buttons. 
Is this aligned with what we see in Figma? (No longer blue, but textPrimary)

Reply:
Yes, it's aligned.